backend/tests/test_project.py

```diff
@@ -71,3 +71,99 @@ def test_project_routes(self):
             data=json.dumps(update_obj),
         )
         self.assertEqual(response.status_code, 404)
+
+    def test_get_all_types(self):
+        test_list = ["dummy type", "dummy type 2"]
+        p1 = Project(
+            address="dummy address",
+            city="dummy city",
+            province="dummy province",
+            postal_code="dummy postal_code",
+            neighbourhood="dummy neighbourhood",
+            year=2020,
+            name="dummy name",
+            type=test_list[0],
+            contacts=[],
+        )
+        p2 = Project(
+            address="dummy address",
+            city="dummy city",
+            province="dummy province",
+            postal_code="dummy postal_code",
+            neighbourhood="dummy neighbourhood",
+            year=2020,
+            name="dummy name",
+            type=test_list[1],
+            contacts=[],
+        )
+        db.session.add(p1)
+        db.session.add(p2)
```

Comment:
@peterghrong If we're going to add more than one, try using `add_all`
https://docs.sqlalchemy.org/en/13/orm/session_api.html#sqlalchemy.orm.session.Session.add_all

Reply:
@peterghrong extra bump